packages/components/src/components/DiagramSequence.vue

```diff
@@ -153,52 +159,64 @@ export default {
       }
       return result;
     },
+    collapsedActions() {
+      return this.collapsedActionState;
+    },
     diagramSpec(): DiagramSpec {
-      const result = new DiagramSpec(this.diagram);
-      this.collapsedActions = []; // eslint-disable-line vue/no-side-effects-in-computed-properties
+      this.collapsedActionState = this.collapsedActionState ?? [];
```

Comment:
The [build](https://app.travis-ci.com/github/getappmap/appmap-js/jobs/610404554) is currently failing because the linter is throwing an error:

```
➤ YN0000: [@appland/components]: /home/travis/build/getappmap/appmap-js/packages/components/src/components/DiagramSequence.vue
➤ YN0000: [@appland/components]:   166:7  error  Unexpected side effect in "diagramSpec" computed property  vue/no-side-effects-in-computed-properties
➤ YN0000: [@appland/components]: 
➤ YN0000: [@appland/components]: ✖ 1 problem (1 error, 0 warnings)
➤ YN0000: [@appland/components]: 
➤ YN0000: [@appland/components]: Process exited (exit code 1), completed in 12s 328ms
```

It looks like you'll need to add `// eslint-disable-lin vue/no-side-effects-in-computed-properties` back to this line.